iot_config/src/route.rs

```diff
@@ -523,18 +523,22 @@ pub fn list_devaddr_ranges_for_route<'a>(
 
 pub fn active_route_stream<'a>(
     db: impl sqlx::PgExecutor<'a> + 'a,
-) -> impl Stream<Item = Route> + 'a {
-    sqlx::query_as::<_, StorageRoute>(
+    since: &'a DateTime<Utc>,
```

Comment:
`active_route_stream` probably needs a rename now that you're not streaming exclusively "active" routes. when i named the function, it was only pulling routes that were "active = true" for orgs that were "locked = false" and obviously couldn't stream deleted routes but i don't think any of those restrictions apply now

Reply:
Addressed in 1390ffc